src/en/reference-hook-tools.md

```diff
@@ -527,19 +527,24 @@ the current workload status. Without arguments, it just prints the workload
 status value e.g. 'maintenance'. With `--include-data` specified, it prints
 YAML which contains the status value plus any data associated with the status.
 
-python:  
+python:
 ```python
 from charmhelpers.core.hookenv import status_get
 
 charm_status = status_get()
 ```
-bash:  
+bash:
 ```bash
 status-get
 
 status-get --include-data
 ```
 
+Use `--service` to get the overall status for the service to which the unit
```

Comment:
If the help text shows --service then it's a bug as we've completely renamed service to application. We support --service so that we don't break existing charms that have not had a chance to update, but our visible face (including documentation) needs to be the one true way forward with is --application. 

This should be updated to --application and anywhere you see service in the user facing juju output needs to have a bug filed against it so that it gets corrected.